eventbusk/bus.py

```diff
@@ -216,9 +224,25 @@ def wrapper() -> None:
                             msg_value = message.value().decode("utf-8")  # type: ignore
                             event_data = json.loads(msg_value)
 
+                            if "event_id" in event_data:
+                                try:
+                                    event_id = uuid.UUID(event_data.pop('event_id'))
+                                except ValueError:
+                                    logger.exception(
+                                        (
+                                            "Error while converting str -> UUID "
+                                        ),
+                                        extra={**log_context, **{"data": event_data}},
+                                        exc_info=True,
+                                    )
+                                    pass
+                            else:
+                                event_id = None
+
                             # TODO: Fix following
                             # Too many arguments for "Event"  [call-arg]
                             event = event_type(**event_data)  # type: ignore
+                            setattr(event, 'event_id', event_id)
```

Comment:
Had to use `setattr` here instead because `event_id` field has `default_factory=uuid.uuid4, init=False`